pkg/network/cache/filesystem.go

```diff
@@ -0,0 +1,41 @@
+package cache
```

Comment:
We don't need an interface here since the implementation is the same but with a pair of constructors, also right now we just need a few filesystems calls, one thing we can do is to move the filesystem.go from this PR somewhere (but not util/comon/helpers). and if it's used at other places add the filesystems calls needed.

Reply:
Hm, I would really have preferred to just copy it over. :/

Reply:
I am fine with not copying the `filesystem.go` file if we don't want  the interface. `defaultfs.go` should be good enough.

Reply:
I am creating `pkg/os/fs` with modified version of https://github.com/dims/kubernetes/tree/master/pkg/util/filesystem so we don't add more random stuff under `pkg/util`.

Also will keep the interface there maybe in the future we need a memory fs for some reason.